Signal/src/views/AudioProgressView.swift

```diff
@@ -73,6 +73,10 @@ import SignalServiceKit
     internal func updateContent() {
         SwiftAssertIsOnMainThread(#function)
 
+        // Prevent the shape layer from animating changes.
+        CATransaction.begin()
+        CATransaction.setValue(kCFBooleanTrue, forKey: kCATransactionDisableActions)
+
```

Comment:
Take it or leave it, but since the start and end of the transaction are getting pretty far apart, it might be more coherent to introduce a block based api like:

```
CATransaction.ows_withoutAnimation {

}
```

Reply:
I'm going to leave it for now, but it's not a bad idea.  It might make more sense as a method on CALayer.